types/chrome/index.d.ts

```diff
@@ -7189,7 +7190,7 @@ declare namespace chrome.webRequest {
          */
         types?: string[];
         /** A list of URLs or URL patterns. Requests that cannot match any of the URLs will be filtered out. */
-        urls: string[] | RegExp[] | "<all_urls>";
+        urls: string[];
```

Comment:
It should be 'url' not 'urls'.
See: https://developer.chrome.com/extensions/events